app/src/main/res/values/strings.xml

```diff
@@ -1022,9 +1022,9 @@ If there are no signs along the whole street which apply for the highlighted sec
     <string name="quest_memorialType_plaque">Plaque</string>
     <string name="quest_memorialType_war_memorial">War memorial</string>
     <string name="quest_memorialType_stone">Stone</string>
-    <string name="quest_memorialType_obelisk">Obelisk</string>
+    <string name="quest_memorialType_obelisk">Obelisk (pointy pillar)</string>
```

Comment:
I still think that "pointy pillar" is not a helpful description of obelisk

Reply:
As no comments on wikipedia yet, perhaps `Obelisk (usually 4-sided)`? Or some other idea?

Reply:
I've decided to not change texts for now; they can always be changed later if it turns out it is still confusing